src/shared/ui/form/form-label.tsx

```diff
@@ -0,0 +1,14 @@
+import { cn } from '../../utils'
+
+import type { LabelHTMLAttributes } from 'react'
+
+interface Props extends LabelHTMLAttributes<HTMLLabelElement> {
+  name: string
+}
+export default function FormLabel({ name, children, className, ...rest }: Props) {
```

Comment:
```suggestion
export default function FormLabel({ name, children, className, ...restProps }: Props) {
```
컴포넌트는 props 객체를 받으니 restProps가 더 명확한것같아요!